main.py

```
@@ -16,8 +17,15 @@
     print("Initializing UsersService")
     users_service = UsersService(databaseWorker)
 
+    bot_token = os.getenv(token)
```

Ruff (0.8.2)

Undefined name `token`

(F821)